### PR TITLE
Live Testing update - App Management

### PR DESCRIPTION
**Pull request by aoleszyn**
### Description
Live Testing update - App Management

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

File: docs/mobile-apps/live-testing/live-mobile-app-testing.md [hunk 1]
```diff
@@ -24,30 +24,30 @@ You can upload your app via the Sauce Labs UI or via the REST API. For informati
...
+<img src={useBaseUrl('img/live-testing/live-mobile-app-management.png')} alt="Upload an app" width="650"/>
 
 ### Deleting an App
 
-Deleting an app in Sauce Labs will delete the whole app (i.e., the group of builds belonging to the same app package). Files associated with app identifiers (i.e., belonging to the same platform and accessible to the same team) are indicated by the + symbol next to version number. Also, the version number shown is the most recently updated file, not necessarily the latest version of the app.
+Deleting an app in Sauce Labs will delete the whole app (i.e., the group of builds belonging to the same app package). Files associated with app identifiers (i.e., belonging to the same platform and accessible to the same team) are indicated by the + symbol next to the version number. Also, the version number shown is the most recently updated file, not necessarily the latest version of the app.
 
-To delete an app, on the Mobile App test page, hover over the test and then click **Delete**.
+To delete an app, on the App Management page, hover over the build and then click **Delete**.
```
Comment (lysannep) [inline docs/mobile-apps/live-testing/live-mobile-app-testing.md:38]:
```suggestion
To delete an app, on the **App Management** page, hover over the build and then click **Delete**.
```

File: docs/mobile-apps/live-testing/live-mobile-app-testing.md [hunk 2]
```diff
@@ -24,30 +24,30 @@ You can upload your app via the Sauce Labs UI or via the REST API. For informati
...
 
-<img src={useBaseUrl('img/live-testing/live-mobile-app-delete.png')} alt="Delete an app" width="650"/>
+<img src={useBaseUrl('img/live-testing/live-mobile-app-management-delete.png')} alt="Delete an app" width="850"/>
 
 ### App Settings
 
-To view or change the app settings, on the Mobile App test page, hover over the app and then click **Settings**.
+To view or change the app settings, on the App Management test page, hover over the app and then click **Settings**.
```
Comment (lysannep) [inline docs/mobile-apps/live-testing/live-mobile-app-testing.md:44]:
```suggestion
To view or change the app settings, on the **App Management** page, hover over the app and then click **Settings**.
```

Comment (lysannep) [reply docs/mobile-apps/live-testing/live-mobile-app-testing.md:44]: The step says to click "Settings" but the tooltip says "Settings and App Versions"

File: docs/mobile-apps/live-testing/live-mobile-app-testing.md [hunk 3]
```diff
@@ -61,7 +61,7 @@ To view your recent configurations, click **Recents**.
 
 | Setting | Description |
 | :--- | :--- |
-| Device Language | Use the dropdown to select the device language. The language selector will tell your application that the locale of the device and region is set to the selected parameter. You wont need to change the language of the OS manually during a session inside iOS/Android settings. ([Read more about Locale here](https://developer.apple.com/documentation/foundation/locale)).  |
+| Device Language | Use the dropdown to select the device language. The language selector will tell your application that the locale of the device and region is set to the selected parameter. You won't need to change the language of the OS manually during a session inside iOS/Android settings. ([Read more about Locale here](https://developer.apple.com/documentation/foundation/locale)).  |
```
Comment (lysannep) [inline docs/mobile-apps/live-testing/live-mobile-app-testing.md:64]:
```suggestion
| Device Language | Use the dropdown to select the device language. The language selector will tell your application that the locale of the device and region is set to the selected parameter. You won't need to change the language of the OS manually during a session inside iOS/Android settings. For more information about the locale setting, see the documentation for [iOS](https://developer.apple.com/documentation/foundation/locale) and [Android](https://developer.android.com/reference/java/util/Locale).  |
```

File: docs/mobile-apps/live-testing/live-mobile-app-testing.md [hunk 4]
```diff
@@ -167,12 +167,13 @@ If you upload an app that is signed with an enterprise certificate, and **Instru
...
-| <img src={useBaseUrl('img/live-testing/more-device-options-icon.png')} alt="More Device Options icon" width="35"/> | More Device Options | **Set Location** - Set the GPS location using coordinates or by dropping a pin on the map. <br/> **Camera Injection** - Opens the **Camera Injection** window. See [Camera Image Injection](/mobile-apps/features/camera-image-injection) for more information.<br/> **Biometric Injection** - Opens the **Biometric Authentication** window. |
+| <img src={useBaseUrl('img/live-testing/more-device-options-icon.png')} alt="More Device Options icon" width="35"/> | More Device Options | **Set Location** - Set the GPS location using coordinates or by dropping a pin on the map. <br/> **Camera Injection** - Opens the **Camera Injection** window. See [Camera Image Injection](/mobile-apps/features/camera-image-injection) for more information.<br/> **Biometric Injection** - Opens the [Biometric Authentication](https://docs.saucelabs.com/mobile-apps/features/biometric-authentication/) window. |
```
Comment (lysannep) [inline docs/mobile-apps/live-testing/live-mobile-app-testing.md:170]:
```suggestion
| <img src={useBaseUrl('img/live-testing/more-device-options-icon.png')} alt="More Device Options icon" width="35"/> | More Device Options | **Set Location** - Set the GPS location using coordinates or by dropping a pin on the map. <br/> **Camera Injection** - Opens the **Camera Injection** window. See [Camera Image Injection](/mobile-apps/features/camera-image-injection) for more information.<br/> **Biometric Injection** - Opens the [Biometric Authentication](/mobile-apps/features/biometric-authentication) window. |
```

File: docs/mobile-apps/live-testing/live-mobile-app-testing.md [hunk 5]
```diff
@@ -78,26 +78,26 @@ Any changes you make to the app settings will affect all uploaded versions of th
...
+On the **App Management** page, hover over the app you want to test and then click **Start Test**.
 
-<img src={useBaseUrl('img/live-testing/live-mobile-app-choose-device.png')} alt="Choose a device" width="650"/>
+<img src={useBaseUrl('img/live-testing/live-mobile-app-management-test.png')} alt="Choose a device" width="750"/>
 
 The device selection page will open, with the option to test on a real device or a virtual device.
 
 :::note
-If you are testing an iOS app, the **Virtual Devices** tab will only appear if the app is configured for simulators.
+If you are testing an IOS app, the device selection will only display the type (real/virtual) configured on the app.
```
Comment (lysannep) [inline docs/mobile-apps/live-testing/live-mobile-app-testing.md:100]:
```suggestion
If you are testing an iOS app, the device selection will only display the type (real/virtual) configured on the app.
```

**Review by lysannep — APPROVED** [review]

File: docs/mobile-apps/live-testing/live-mobile-app-testing.md [hunk 6]
```diff
@@ -24,30 +24,30 @@ You can upload your app via the Sauce Labs UI or via the REST API. For informati
 
 To upload an app via the Sauce Labs UI:
 
-1. On Sauce Labs, in the left panel, click **LIVE** and then click **Mobile App**.
-2. Click **Upload App**. You can either drag and drop an app, or browse for and select the file. We currently support \*.apk Android app files, \*.aab Android App Bundle files and \*.ipa or \*.zip iOS app files (\*.zip files are parsed to determine whether a valid \*.app bundle exists). Non-app file uploads are not supported in the UI at this time, but can be uploaded through the API.
+1. On Sauce Labs, in the left panel, click **App Management**.
+2. To upload an app you can either drag and drop an app or browse for and select the file. We currently support \*.apk Android app files, \*.aab Android App Bundle files and \*.ipa or \*.zip iOS app files (\*.zip files are parsed to determine whether a valid \*.app bundle exists). Non-app file uploads are not supported in the UI at this time, but can be uploaded through the API.
```
Comment (aliciakrebs) [inline docs/mobile-apps/live-testing/live-mobile-app-testing.md:28]:
```suggestion
2. To upload an app you can either drag and drop an app or browse for and select the file. We currently support \*.apk Android app files, \*.aab Android App Bundle files, and \*.ipa or \*.zip iOS app files (\*.zip files are parsed to determine whether a valid \*.app bundle exists). Non-app file uploads are not supported in the UI at this time, but can be uploaded through the API.
```

File: docs/mobile-apps/live-testing/live-mobile-app-testing.md [hunk 7]
```diff
@@ -24,30 +24,30 @@ You can upload your app via the Sauce Labs UI or via the REST API. For informati
...
+<img src={useBaseUrl('img/live-testing/live-mobile-app-management.png')} alt="Upload an app" width="650"/>
 
 ### Deleting an App
 
-Deleting an app in Sauce Labs will delete the whole app (i.e., the group of builds belonging to the same app package). Files associated with app identifiers (i.e., belonging to the same platform and accessible to the same team) are indicated by the + symbol next to version number. Also, the version number shown is the most recently updated file, not necessarily the latest version of the app.
+Deleting an app in Sauce Labs will delete the whole app (i.e., the group of builds belonging to the same app package). Files associated with app identifiers (i.e., belonging to the same platform and accessible to the same team) are indicated by the + symbol next to the version number. Also, the version number shown is the most recently updated file, not necessarily the latest version of the app.
 
-To delete an app, on the Mobile App test page, hover over the test and then click **Delete**.
+To delete an app, on the **App Management** page, hover over the build and then click **Delete**.
```
Comment (aliciakrebs) [inline docs/mobile-apps/live-testing/live-mobile-app-testing.md:38]:
```suggestion
To delete an app, on the **App Management** page, hover over the app and then click **Delete**.
```

**Review by aliciakrebs — APPROVED** [review]

**Pull request merged**